src/ExpoChannel.php

```diff
@@ -55,20 +29,14 @@ public function send($notifiable, Notification $notification)
                 $notification->toExpoPush($notifiable)->toArray(),
                 config('exponent-push-notifications.debug')
             );
-        } catch (ExpoException $e) {
+        } catch (CouldNotSendNotification $e) {
```

Comment:
not sure we should change the excetion. Accordingly to the phpdoc this method may throw `CouldNotSendNotification` but your version would never throw it